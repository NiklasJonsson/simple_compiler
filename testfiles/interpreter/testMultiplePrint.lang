int main() {
	print(4+5);
	print(9*19);
	print(78+3);
}