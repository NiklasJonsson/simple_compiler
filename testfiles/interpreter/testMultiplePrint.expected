9
171
81
